lib/sqlalchemy/sql/selectable.py

```diff
@@ -113,6 +113,7 @@
 
 
 if TYPE_CHECKING:
+    from sqlalchemy.orm import DeclarativeBase
```

Comment:
this isn't needed (also we can't have ORM imports in core in any case, DeclarativeBase is too narrow as well)